components/specification/publish

```diff
@@ -153,18 +153,23 @@ contains() {
 }
 
 devmarker="dev"
-root=published
-
+root=${1:-published}
+root=${root%/}
 
 # Clean up
 
-rm -rf "$root"
+rm -rf "$root/*"
```

Comment:
Why not clean up the root directory properly?  The glob can leave junk around which it doesn't match.  Better to remove entirely and recreate as on the next line.  Why was this changed?